gobblin-utility/src/main/java/gobblin/broker/iface/ScopeType.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.broker.iface;
+
+import java.util.Collection;
+
+import javax.annotation.Nullable;
+
+
+/**
+ * Represents a DAG of scope types.
+ *
+ * For example, the topology of a distributed application might have scopes as follows:
+ * GLOBAL -> INSTANCE --> JOB       --> TASK
```

Comment:
nit: It is useful to enclose that in a &lt;pre&gt; tag otherwise, HTML will mess up the alignment.